src/BloomExe/FontProcessing/FontMetadata.cs

```diff
@@ -0,0 +1,351 @@
+					case FontEmbeddingRight.EditableButNoSubsetting:
+						fsType = "Editable";
+						break;
+					case FontEmbeddingRight.Installable:
+					case FontEmbeddingRight.InstallableButNoSubsetting:
+						fsType = "Installable";
+						break;
+					case FontEmbeddingRight.PreviewAndPrint:
+					case FontEmbeddingRight.PreviewAndPrintButNoSubsetting:
+						fsType = "Print and preview";
+						break;
+					case FontEmbeddingRight.RestrictedLicense:
+						fsType = "Restricted License";
+						break;
+					case FontEmbeddingRight.EditableButNoSubsettingAndWithBitmapsOnly:
+					case FontEmbeddingRight.EditableButWithBitmapsOnly:
+					case FontEmbeddingRight.InstallableButNoSubsettingAndWithBitmapsOnly:
+					case FontEmbeddingRight.InstallableButWithBitmapsOnly:
+					case FontEmbeddingRight.PreviewAndPrintButNoSubsettingAndWithBitmapsOnly:
+					case FontEmbeddingRight.PreviewAndPrintButWithBitmapsOnly:
+						fsType = "Bitmaps Only";
+						break;
+				}
+				designer = gtf.DesignerNames[english];
+				designerURL = gtf.DesignerUrls[english];
+				license = gtf.LicenseDescriptions[english];
+				manufacturer = gtf.ManufacturerNames[english];
+				manufacturerURL = gtf.VendorUrls[english];
+				trademark = gtf.Trademarks[english];
+			}
+			catch (Exception)
+			{
+				// file is somehow corrupt or not really a font file? Just ignore it.
+			}
+#endif
+			variants = group.GetAvailableVariants().ToArray();
+
+			// Now for the hard part: setting DeterminedSuitability
+			if (!String.IsNullOrEmpty(license))
+			{
+				if (license.Contains("Open Font License") || license.Contains("OFL") ||
+					license.StartsWith("Licensed under the Apache License") ||
+					license.Contains("GNU GPL") || license.Contains("GNU General Public License") || license.Contains(" GPL ") || license.Contains(" GNU ") ||
+					(license.Contains("GNU license") && license.Contains("www.gnu.org")) ||
+					license.Contains("GNU LGPL") || license.Contains("GNU Lesser General Public License"))
+				{
+					determinedSuitability = "ok";
+					if (license.Contains("Open Font License") || license.Contains("OFL"))
+						determinedSuitabilityNotes = "Open Font License";
+					else if (license.StartsWith("Licensed under the Apache License"))
+						determinedSuitabilityNotes = "Apache";
+					else if (license.Contains("GNU LGPL") || license.Contains("GNU Lesser General Public License"))
+						determinedSuitabilityNotes = "GNU LGPL";
+					else
+						determinedSuitabilityNotes = "GNU GPL";
+					return;
+				}
+				if (license.Replace('\n',' ').Contains("Permission is hereby granted, free of charge, to any person obtaining a copy of the fonts accompanying this license (\"Fonts\") and associated documentation files (the \"Font Software\"), to reproduce and distribute the Font Software, including without limitation the rights to use, copy, merge, publish, distribute, and/or sell copies of the Font Software, and to permit persons to whom the Font Software is furnished to do so") &&
```

Comment:
Feels a bit brave to require the exact wording? Would "free of charge" be enough?